code/modules/mob/living/simple_animal/abnormality/he/golden_false_apple.dm

```diff
@@ -0,0 +1,392 @@
+			to_chat(user, "<span class='nicegreen'>Your body is engulfed with a warm glow, numbing your injuries.</span>")
+		else
+			to_chat(user, "<span class='nicegreen'>The glow surrounding your body brightens.</span>")
+			G.add_stacks(1)
+			G.refresh()
+			playsound(src, 'sound/abnormalities/goldenapple/Gold_Sparkle.ogg', 100, 1)
+	else
+		datum_reference.qliphoth_change(-1)
+	return
+
+/mob/living/simple_animal/hostile/abnormality/golden_apple/FailureEffect(mob/living/carbon/human/user, work_type, pe)
+	datum_reference.qliphoth_change(-1)
+	return
+
+/mob/living/simple_animal/hostile/abnormality/golden_apple/PostWorkEffect(mob/living/carbon/human/user, work_type, pe)
+	..()
+	var/datum/status_effect/stacking/golden_sheen/G = user.has_status_effect(/datum/status_effect/stacking/golden_sheen)
+	if (G.stacks >= 3)//Kills the employee if you already have 2 stacks of golden sheen and instantly breaches in phase 2
+		user.gib()
+		datum_reference.qliphoth_change(-2)
+		Become_Rotten()
+		var/mob/living/carbon/human/H = user
+		addtimer(CALLBACK(src, .proc/Digest_Person, H), 5 SECONDS)//becomes its "berserk" form; the user is assimilated into it
+
+//***Breach Mechanics***//
+/mob/living/simple_animal/hostile/abnormality/golden_apple/BreachEffect(mob/living/carbon/human/user)
+	..()
+	icon_state = "gold_apple"
+	icon_living = "gold_apple"
+
+//switch to the second form if var is_maggot is 0
+/mob/living/simple_animal/hostile/abnormality/golden_apple/death()
+	if(health > 0)
+		return
+	if(!is_maggot)
+		playsound(src, 'sound/abnormalities/goldenapple/Gold_Attack.ogg', 100, 1)
+		addtimer(CALLBACK(src, .proc/Eat_Employees), 10 SECONDS)
+		addtimer(CALLBACK(src, .proc/Become_Rotten), 15 SECONDS)
+		return ..()
+	density = FALSE
+	playsound(src, 'sound/abnormalities/goldenapple/False_Dead.ogg', 100, 1)
+	animate(src, alpha = 0, time = 10 SECONDS)
+	QDEL_IN(src, 10 SECONDS)
+	..()
+
+/mob/living/simple_animal/hostile/abnormality/golden_apple/proc/Become_Rotten()//phase 2
+	if(QDELETED(src))
+		return
+	playsound(src, 'sound/abnormalities/goldenapple/Gold_Falsify.ogg', 75, 1)
+	icon = 'ModularTegustation/Teguicons/96x48.dmi'
+	icon_state = "false_apple"
+	icon_living = "false_apple"
+	icon_dead = "false_dead"
+	name = "False Apple"
+	desc = "The apple ruptured and a swarm of maggots crawled inside, metamorphosing into a hideous face."
+	pixel_x = -48
+	pixel_y = 0
+	light_range = 0
+	light_power = 0
+	maxHealth = 1200
+	health = 1200
+	attack_sound = list('sound/abnormalities/goldenapple/False_Attack.ogg', 'sound/abnormalities/goldenapple/False_Attack2.ogg')
+	melee_damage_lower = 30
+	melee_damage_upper = 45
+	attack_verb_continuous = "pummels"
+	attack_verb_simple = "pummel"
+	move_to_delay = 3
+	damage_coeff = list(BRUTE = 1, RED_DAMAGE = 1.5, WHITE_DAMAGE = 1.5, BLACK_DAMAGE = 1, PALE_DAMAGE = 0.5)
+	melee_damage_type = BLACK_DAMAGE
+	fear_level = WAW_LEVEL
+	revive(full_heal = TRUE, admin_revive = FALSE)
+	++is_maggot
```

Comment:
This should be a boolean not simply incrementing a number, once again for code clarity.